src/codeu/chat/server/Server.java

```diff
@@ -172,6 +176,24 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       }
     });
 
+    //Gets the Server information
+    this.commands.put(NetworkCode.SERVER_VERSION_REQUEST, new Command(){
+      @Override
+      public void onMessage(InputStream in, OutputStream out) throws IOException {
+
```

Comment:
In this file, there is also an extra blank line in the previous method, line 169 and 170. Would you still want me to remove the extra blank line? or should i change it to reflect line 191 and 192 where the extra blank line was removed?